.github/CODEOWNERS

```diff
@@ -322,6 +322,9 @@ x-pack/examples/files_example @elastic/kibana-app-services
 /x-pack/plugins/actions/ @elastic/response-ops
 /x-pack/plugins/event_log/ @elastic/response-ops
 /x-pack/plugins/task_manager/ @elastic/response-ops
+/x-pack/plugins/stack_connectors/ @elastic/response-ops
+/x-pack/plugins/stack_connectors/server/connector_types/stack/ @elastic/response-ops-execution
+/x-pack/plugins/stack_connectors/server/connector_types/cases/ @elastic/response-ops-cases
```

Comment:
@ymao1 How does this work? Can someone from the `@elastic/response-ops` team approve a PR that touches a file in `x-pack/plugins/stack_connectors/server/connector_types/cases`?

Reply:
I believe this will ping the individual teams if those subdirectories are touched but anyone from response ops can approve anything within `stack_connectors`. I wanted response ops to own the top level plugin to be pinged on changes that are outside of the specific connector types and wasn't sure how else to do that. What do you think?

Reply:
This is what I wanted to confirm. Yes, the response ops team should be able to approve anything in the `stack_connectors` (including all connector types). Otherwise, PRs can be blocked or delayed because someone from the sub-teams needs to approve. Also, it does not necessarily mean that the Cases team owns connectors that belong to the`connector_types/cases` folder. So the current behavior aligns with that assumption.